electives/18623.md

```diff
@@ -0,0 +1,66 @@
+# 18-623: Analog Integrated Circuit Design
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| HW       | 7          |
+| Projects | 8          |
+| Exams    | 6          |
+
+This class provides an overview of analog integrated circuits, covering a number of IC designs and techniques at the transistor-level.
+
+It covers MOSFET modes of operation, as well as its various applications. The class then shows a number of amplifier topologies and their use cases. It also covers the differential pair and the operational amplifier in great detail, among other topologies. It focuses on how to analyze and design analog circuits to meet certain specifications and constraints, showing the tradeoff between these goals.
+
+Please note that this guide was written from Fall 2021; the assignments and material for the course may change slightly in future offerings of the course.
+
+## Homeworks
+
+You will have around 5-6 homeworks throughout the semester. They involve both conceptual questions which require you to work through circuits and calculations on paper, as well as some simulation-based problems. The homeworks can be challenging but, as with most classes, be sure to start early and utilize office hours. The textbook may also be very helpful to review relevant concepts, although it can be quite dense in its analysis.
+
+The TAs will hold lab sessions on Fridays, some of which are dedicated to helping you out with the homework. Be sure to attend those, and come with lots of questions! That is a great time to get some clarification for the homework, as well as lecture concepts in general.
+
+Ultimately, the homeworks have a lower weight on your grade, relative to the projects and exams. However, you should still give a good effort on them - be sure to go back and learn what you did not understand fully when doing the homework. This will help you when it comes time to tackle the projects and exams.
```

Comment:
```suggestion
Ultimately, the homeworks have a lower weight on your grade, relative to the projects and exams. However, you should still give a good effort on them -- be sure to go back and learn what you did not understand fully when doing the homework. This will help you when it comes time to tackle the projects and exams.
```